dm/pkg/binlog/event/util_test.go

```diff
@@ -17,24 +17,27 @@
 package event
 
 import (
+	"github.com/stretchr/testify/require"
```

Comment:
configurate the IDE to group and sort the import, like

https://www.jetbrains.com/help/go/creating-and-optimizing-imports.html#use-the-goimports-local-grouping